src/api/codeforces/interfaces.ts

```diff
@@ -0,0 +1,13 @@
+export interface UpcomingContestResponseSchema {
+    id: number;
+    name: string;
+    type: string;
+    phase: string;
+    frozen: boolean;
+    durationSeconds: number;
+    startTimeSeconds: number;
+    relativeTimeSeconds: number;
+}
```

Comment:
Change the name from `UpcomingContestResponseSchema` to `ContestResponseSchema` since this schema is common for the Codeforces API route for contests